interactive_ai/workflows/train/trainer/pyproject.toml

```diff
@@ -17,7 +17,7 @@ dependencies = [
 [tool.uv.sources]
 geti-kafka-tools = { path = "../../../../libs/kafka_tools" }
 geti-telemetry-tools = { path = "../../../../libs/telemetry_tools" }
-otx = { git = "https://github.com/open-edge-platform/training_extensions.git", branch = "release/2.5" }
+otx = { git = "https://github.com/open-edge-platform/training_extensions.git", branch = "geti-classic" }
```

Comment:
For reference, this change partially addresses https://github.com/open-edge-platform/geti/issues/1157
The other part of the ticket - updating the reference in the Docker image to download templates - can be done in another PR